packages/labs/ssr/src/test/integration/tests/basic.ts

```diff
@@ -42,7 +42,7 @@ import {
 
 import {anyHtml, SSRTest, SSRTestDescription} from './ssr-test.js';
 import {AsyncDirective} from 'lit/async-directive.js';
-import {serverhtml} from '../../../lib/server-template.js';
+import {html} from '../../../lib/server-template.js';
```

Comment:
On line 9 there is an import `html` from 'lit'. I think this should be `{html as serverhtml}`

Reply:
Ah, yep. Fixed